### PR TITLE
 introduce Asset, VideoAsset/ImageAsset, project.elevatorPitch config

### LINKED ISSUE #75: proposal: add project.media and project.elevatorPitch
https://github.com/ubclaunchpad/new/pull/74#issuecomment-618788990

* `project.media = Video | Image`: media to feature instead of the banner inside a project modal. Similarly, I'm thinking of renaming `project.images` into just `project.banner = { uri, hasName }`
* `project.elevatorPitch = string`: as implied by name
* I'll update each `project.description` to be even shorter as well ( <5 words seems like a good threshold for the idea)

### PR DESCRIPTION
**Pull request by bobheadxi**
Implements proposed spec in #75 

Won't be updating content in this PR - we can do that piecemeal as we go, but this should get us set up.

### CODE REVIEW

File: src/components/TeamProjectCard.vue [hunk 1]
```diff
@@ -3,14 +3,14 @@
     <div
       class="project-image"
       :class="{
-        'name-on-hover': !team.project.images.bannerHasName
+        'name-on-hover': !team.project.banner.hasName
       }"
       :style="{
-        'background-image': 'url(' + team.project.images.bannerURI + ')',
+        'background-image': 'url(' + team.project.banner.url + ')',
```
Comment (RachitMalik12) [inline src/components/TeamProjectCard.vue:9]: nit: can we use template strings here i.e. `` `some ${variable}` ``

**Review by RachitMalik12 — APPROVED** [review]
Looks good! Just to clarify elevator pitch is a summary description like the more detailed description on the old launchpad website?

File: src/components/TeamProjectCard.vue (hunk 1 above)
Comment (bobheadxi) [reply src/components/TeamProjectCard.vue:9]: unfortunately you cannot use es6 template strings inside vue bindings AFAIK: https://github.com/vuejs/vue/issues/2364

**Comment by bobheadxi** [comment]
> elevator pitch is a summary description like the more detailed description on the old launchpad website?

yep! but probably not as long as those ones :)

**Comment by bobheadxi** [comment]
@RachitMalik12 @srijonsaha  threw up a question on `#tm-website-revamp` about the config - do you mind dropping in the thread?

**Comment by RachitMalik12** [comment]
> @RachitMalik12 @srijonsaha threw up a question on `#tm-website-revamp` about the config - do you mind dropping in the thread?

 I chimed in there..

**Pull request merged**